### PR TITLE
feat: Allow unittest-version be set to 'latest' or empty string

### PR DESCRIPTION
**Pull request by camalot**
Allow the `unittest-version` property to be either an empty string, or `latest`. 

This addresses #15

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -65,6 +65,10 @@ runs:
           fi
         fi
         helm_ut_plugin_version="${{ inputs.unittest-version }}"
+        # if helm_ut_plugin_version is "latest" set it to empty string
+        if [ "$helm_ut_plugin_version" = "latest" ]; then
+          helm_ut_plugin_version=""
+        fi
         if [ -n "$helm_ut_plugin_version" ]; then
```
Comment (d3adb5) [inline action.yml]: The `$helm_ut_plugin_version` variable is only used here in this script, so I make the following style suggestion:

```suggestion
        if [ -n "$helm_ut_plugin_version" ] && [ "$helm_ut_plugin_version" != "latest" ]; then
```

**Review by d3adb5 — APPROVED** [review]
Thank you so much for this contribution, @camalot! I left a small suggestion, let me know what you think of it. Overall this looks good to me, so I'll leave an approval and wait for your feedback.

File: action.yml (hunk 1 above)
Comment (camalot) [reply action.yml]: Yeah, I was going back and forth on which way to go with this.

**Comment by camalot** [comment]
I think i squashed the commits correctly... if not, let me know.

**Comment by d3adb5** [comment]
There were suddenly 4 commits in this PR, so I think something went wrong on your local worktree. Nonetheless, I squashed and merged directly through GitHub. I'll make a release for this. Thank you, @camalot!

**Pull request merged**